src/types/helpers.ts

```diff
@@ -4,7 +4,10 @@ import { UnknownAction } from '../internal/types';
 
 export type ActionStream = Observable<UnknownAction>;
 
-export type ActionDispatcher = (action: UnknownAction) => void;
+export type ActionDispatcher = (
+  action: UnknownAction,
+  namespace?: string
```

Comment:
Should we not use well defined namespaces, e.g use an `enum Namespace` ?

Reply:
`string` allows us to pass string enums. This means we can keep RxBeach separate from ardoq-front, with no need for it to know the namespaces we use in ardoq-front. In ardoq-front we can define a string enum if we want.